### PR TITLE
Add datastore feature to store comments.

### PR DESCRIPTION
**Pull request by asam01**
https://screenshot.googleplex.com/KvSoMJk4MrH.png - initial comment loaded
https://screenshot.googleplex.com/mmxem6M5TNz.png - adding a second comment
https://screenshot.googleplex.com/618pqsyECMk.png - deleting a comment

(sorry for the multiple commits, it wasn't allowing me to revert a commit that I had already pushed - the only difference between the commits is that I removed some commented-out code in the later commit.)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Task.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
```
Comment (yukyhaha) [inline portfolio/src/main/java/com/google/sps/data/Task.java:17]: update the doc here?

Comment (asam01) [reply portfolio/src/main/java/com/google/sps/data/Task.java]: fixed.

File: portfolio/src/main/java/com/google/sps/data/Task.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+package com.google.sps.data;
+
+/** A comment on the forum. */
+public final class Task {
+
+  private final long id;
+  private final String name;
+  private final String comment;
+  private final long timestamp;
```
Comment (yukyhaha) [inline portfolio/src/main/java/com/google/sps/data/Task.java:23]: is the timestamp supposed to be used somewhere in the front end?

Comment (yukyhaha) [reply portfolio/src/main/java/com/google/sps/data/Task.java:23]: ah, i see, it's used to sort the comments

**Pull request merged**